Gemfile

```diff
@@ -17,15 +17,15 @@ gem 'discard', '~> 1.2'
 gem 'discordrb-webhooks', '~> 3.4'
 gem 'dry-initializer', '~> 3.1'
 gem 'friendly_id', '~> 5.5'
-gem 'github_api', '~> 0.19'
+gem 'github_api2', '~> 1.0'
```

Comment:
Hey @JacobMarq, I was just looking around to see what we use this for, it doesn't look like we use it anymore and we can remove it I'm afraid. 

The Git history says it was once used to pull in the curriculum lesson content. But we must have switched to [Octokit](https://github.com/octokit/octokit.rb) for that at some point and forgot to remove this dependency. 

It's probably not much consolation, but thank you so much for going so far out your way to get this working. Hopefully other projects that use still github_api can find your fork and make use of it.